### PR TITLE
Facenet

### PR DESCRIPTION
**Pull request by shivam-raj**
FaceNet Encoder for returning vector embeddings for each face.
Based on FaceNet by : Florian Schroff, Dmitry Kalenichenko, James Philbin ( PyTorch implementation)

### CODE REVIEW

File: hub/executors/encoders/image/facenet/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: torch facenet
+description: an image with the encoder of facenet pytorch implementation
+author: Shivam Raj (connect2shivamraj@gmail.com)
+url: NA
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.3
+vendor: Jina AI Limited
+license: apache-2.0
```
Comment (nan-wang) [inline hub/executors/encoders/image/facenet/manifest.yml:8]: missing `update` key. Would you please have a look at [`update`](https://github.com/jina-ai/jina-hub#schema-of-manifestyml) in the documentation?

Comment (shivam-raj) [reply hub/executors/encoders/image/facenet/manifest.yml:8]: @nan-wang Heyy,
I added the update field.
Should I publish the image to DockerHub myself ?

Comment (nan-wang) [reply hub/executors/encoders/image/facenet/manifest.yml:8]: No worry about that. Our CI/CD process will take care of it. ;)

**Comment by nan-wang** [comment]
@shivam-raj Oops~ It seems that you forget to check out our guidelines on [Commit Style](https://docs.jina.ai/chapters/CONTRIBUTING.html#commit-style). The thing is that we lints the commits with the [`commitlint` ](https://github.com/wagoid/commitlint-github-action). It would be great if you can make a new PR and make sure the commit are in the right format. Please let me know if you meet any problem. Many thanks again!🖖

**Pull request closed without merging**